### PR TITLE
update default net to nn-7e66505906a6.nnue

### PR DESCRIPTION
**Pull request by vondele**
trained with pytorch using the master branch and recommended settings,
the data used is the previous 64B binpack enhanced with a 2B binpack generated using an opening book
of positions for with the static eval is significantly different from d9 search.

book: https://drive.google.com/file/d/1rHcKY5rv34kwku6g89OhnE8Bkfq3UWau/view?usp=sharing
book generation: https://github.com/vondele/Stockfish/commit/3ce43ab0c4ce09c1fc5bca5ca27a248e67fddd24
binpack: https://drive.google.com/file/d/1rHcKY5rv34kwku6g89OhnE8Bkfq3UWau/view?usp=sharing

data generation command:

generate_training_data depth 9 count 31250000 random_multi_pv 2 random_multi_pv_diff 100 random_move_max_ply 8 random_move_count 3 set_recommended_uci_options eval_limit 32000 output_file_name output.binpack book wrongNNUE.epd seed ${RANDOM}${RANDOM}

training command:

python train.py ../../all_d9_fishd9_d8_d10_wrong_shuffle.binpack ../../all_d9_fishd9_d8_d10_wrong_shuffle.binpack  --gpus 1 --threads 2 --num-workers 2 --batch-size 16384 --progress_bar_refresh_rate 300 --smart-fen-skipping --random-fen-skipping 3 --features=HalfKAv2^   --lambda=1.0  --max_epochs=400 --seed $RANDOM --default_root_dir exp/run_5

passed STC
https://tests.stockfishchess.org/tests/view/60b7c79a457376eb8bcaa104
LLR: 2.94 (-2.94,2.94) <-0.50,2.50>
Total: 64592 W: 6254 L: 6028 D: 52310
Ptnml(0-2): 255, 4785, 22020, 4951, 285

passed LTC:
https://tests.stockfishchess.org/tests/view/60b85307457376eb8bcaa182
LLR: 2.96 (-2.94,2.94) <0.50,3.50>
Total: 45560 W: 1998 L: 1826 D: 41736
Ptnml(0-2): 36, 1604, 19335, 1762, 43

Bench: 4364128

### CODE REVIEW

**Comment by snicolet** [comment]
Merged via d53071eff4e75bc77dc86f65c52358d8014cb71c, congrats :-)

**Comment by snicolet** [comment]
How do we set the status of default net to the new net in https://tests.stockfishchess.org/nns, btw?

**Comment by vondele** [comment]
automatic

**Comment by vondele** [comment]
IIRC it checks if the master branch from official-stockfish contains the net, if so the green light is switched on.

**Comment by snicolet** [comment]
OK, maybe there is some lag

**Comment by vondele** [comment]
yes, happens on the first new test that uses that master version.
